client/scss/components/_buttons.scss

```diff
@@ -70,3 +70,11 @@
     border-color: color('java');
   }
 }
+
+.btn--full-width-s {
+  width: 100%;
```

Comment:
possibly a good place to use `@include respond-between('small', 'medium') {}`?

Reply:
Not mobile first though?

Reply:
it's mobile first, but with a cap

Reply:
mobile first = w: 100%
rest = auto?